ansible/roles/pgo-operator/templates/add-targeted-namespace.sh.j2

```diff
@@ -29,17 +29,16 @@ PG_SA="pgo-pg"
 # create the namespace if necessary
 {{ kubectl_or_oc }} get ns {{ item }}  > /dev/null
 if [ $? -eq 0 ]; then
-	echo "namespace" {{ item }} "already exists"
+	echo "namespace" {{ item }} "already exists, adding labels"
+	# set the labels so that existing namespace is owned by this installation
+	{{ kubectl_or_oc }} label namespace/{{ item }} pgo-created-by=add-script
+	{{ kubectl_or_oc }} label namespace/{{ item }} vendor=crunchydata
+	{{ kubectl_or_oc }} label namespace/{{ item }} pgo-installation-name={{ pgo_installation_name }}
 else
 	echo "namespace" {{ item }} "is new"
 	{{ kubectl_or_oc }} create ns {{ item }}
```

Comment:
It looks like this command will create a namespace without any labels.

Reply:
Yeah it does look like that. The bash script uses a `target-namespace.yaml` that includes the labels and i assumed they did the same thing. Would it be better to update the ansible script to use a yaml file or just revert this change for ansible?

Reply:
Reverted this file change. Ansible doesn't suffer from the same false positive error that bash does so changing the ansible version seems unnecessary.